runtime/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 
 // TODO: disable everything else when not using runtime-wasm
 
+#[allow(unused_imports)] // rust seems to be wrong here
```

Comment:
not sure what happened here...

Reply:
It is rust warning about old import method, since we upgraded to edition 2021 we don't need this. The error message not the most helpful it telling us this though.